openshift-knative-operator/cmd/operator/kodata/knative-serving/1.13/1-serving-crds.yaml

```diff
@@ -1366,121 +1370,8 @@ spec:
                     CACerts:
                       description: CACerts is the Certification Authority (CA) certificates in PEM format according to https://www.rfc-editor.org/rfc/rfc7468.
                       type: string
-                    name:
-                      description: Name is the name of the address.
-                      type: string
-                    url:
-                      type: string
-                annotations:
-                  description: Annotations is additional Status fields for the Resource to save some additional State as well as convey more information to the user. This is roughly akin to Annotations on any k8s resource, just the reconciler conveying richer information outwards.
-                  type: object
-                  additionalProperties:
-                    type: string
-                conditions:
-                  description: Conditions the latest available observations of a resource's current state.
-                  type: array
-                  items:
-                    description: 'Condition defines a readiness condition for a Knative resource. See: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties'
-                    type: object
-                    required:
-                      - status
-                      - type
-                    properties:
-                      lastTransitionTime:
-                        description: LastTransitionTime is the last time the condition transitioned from one status to another. We use VolatileTime in place of metav1.Time to exclude this from creating equality.Semantic differences (all other things held constant).
-                        type: string
-                      message:
-                        description: A human readable message indicating details about the transition.
-                        type: string
-                      reason:
-                        description: The reason for the condition's last transition.
-                        type: string
-                      severity:
-                        description: Severity with which to treat failures of this type of condition. When this is not specified, it defaults to Error.
-                        type: string
-                      status:
-                        description: Status of the condition, one of True, False, Unknown.
-                        type: string
-                      type:
-                        description: Type of condition.
-                        type: string
-                observedGeneration:
-                  description: ObservedGeneration is the 'Generation' of the Service that was last processed by the controller.
-                  type: integer
-                  format: int64
-                url:
-                  description: URL is the URL of this DomainMapping.
-                  type: string
-    - name: v1alpha1
```

Comment:
Note: We drop v1alpha1 of DomainMapping here.

Reply:
But this is also in 1.12, so should be fine.

Reply:
You mean 1.11 right?

Reply:
Yes and no :) I meant we'd remove it even if we bump just to 1.12